sysdig/data_source_sysdig_current_user_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build tf_acc_sysdig
+//go:build tf_acc_sysdig || tf_acc_monitor
 
```

Comment:
By specifying `tf_acc_monitor` you are going to run test on IBM which you should not.

Reply:
Very few resources are refactored to use new v2 client. And only they have ability to run on IBM. User resources/data sources right now do not have this feature.